src/RBControl_linesensor.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <vector>
+#include <atomic>
+
+#include <driver/spi_master.h>
+
+namespace rb {
+
+class LineSensor {
+    friend class Manager;
+public:
+    static constexpr int LEDS = 8;
+
+    struct Config {
+        Config(int freq = 1350000, spi_host_device_t spi_dev = HSPI_HOST,
```

Comment:
I would prefer the first pinout settings and then other settings because as a user you will probably change more often the pinout. What do you think @Tasssadar?

Reply:
Sure, why not, you don't have to put them into constructor though you can do this:

```cpp
auto cfg = Config();
cfg.miso = GPIO_NUM_14;
...
```

Whis is clearer, because you can't tell which pin are you setting with `Config(GPIO_NUM_14, GPIO_NUM_15, ..)` - _was the first one miso or mosi?_

Reply:
Yes, ok that makes sense.